download.php

```diff
@@ -1,28 +1,59 @@
 <div class="row">
-	<div class="col-sm-4">
-		<h3>Source</h3>
-		<p>Fork us at <a href="https://github.com/namecoin/namecoin">Github</a>.</p>
-	</div>
-	<div class="col-sm-4">
-		<h3>Windows</h3>
-		<p>
-			Download <a href="https://namecoin.info/files/Namecoin_v0.3.80_setup.exe">v0.3.80 installer</a>.<br>
-			<a target="_blank" href="https://forum.namecoin.info/viewtopic.php?f=8&t=2123">Release information and signatures</a>.
-		</p>
-	</div>
-	<div class="col-sm-4">
-		<h3>Mac OS X</h3>
-		<p>
-			Download <a href="https://namecoin.info/files/Namecoin-Qt.app-0.3.80-a00c33d.zip">v0.3.80 Namecoin-Qt</a>.<br>
-			<a target="_blank" href="https://forum.namecoin.info/viewtopic.php?f=8&t=2235">Release information</a>.
-		</p>
-	</div>
-	<div class="col-sm-4">
-		<h3>Linux Packages</h3>
-		<p>
-			Linux packages provided by OBS.<br>
-			<a href="https://software.opensuse.org/download.html?project=home%3Ap_conrad%3Acoins&amp;package=namecoin">Command line client (namecoind)</a>.<br>
-			<a href="https://software.opensuse.org/download.html?project=home%3Ap_conrad%3Acoins&amp;package=namecoin-gui">Graphical interface (Namecoin-Qt)</a>.
-		</p>
-	</div>
-</div>
\ No newline at end of file
+    <h2>Namecoin-Qt Client</h2>
+        <p>
+            Client for normal use (but not for mining or development). Always wait for six confirmations.
+            <a href="https://github.com/namecoin/namecoin">Source on Github</a>.
+        </p>
+        <div class="col-sm-4">
+            <h3>Windows</h3>
+            <p>
+                Download <a href="https://namecoin.info/files/Namecoin_v0.3.80_setup.exe">v0.3.80 installer</a>.<br>
+                <a target="_blank" href="https://forum.namecoin.info/viewtopic.php?f=8&t=2123">Release information and signatures</a>.
+            </p>
+        </div>
+        <div class="col-sm-4">
+            <h3>Linux Packages</h3>
+            <p>
+                Linux packages provided by OBS.<br>
+                <a href="https://software.opensuse.org/download.html?project=home%3Ap_conrad%3Acoins&amp;package=namecoin">Command line client (namecoind)</a>.<br>
+                <a href="https://software.opensuse.org/download.html?project=home%3Ap_conrad%3Acoins&amp;package=namecoin-gui">Graphical interface (Namecoin-Qt)</a>.
+            </p>
+        </div>
+        <div class="col-sm-4">
+            <h3>Mac OS X</h3>
+            <p>
+                Download <a href="https://namecoin.info/files/Namecoin-Qt.app-0.3.80-a00c33d.zip">v0.3.80 Namecoin-Qt</a>.<br>
+                <a target="_blank" href="https://forum.namecoin.info/viewtopic.php?f=8&t=2235">Release information</a>.
+            </p>
+        </div>
+</div>
+<div class="row">
+    <h2>NMControl Middleware</h2>
+        <p>
+            NMControl connects your browser/application to the client (or soon to an API). It allows you, for example, to browse .bit domains.
```

Comment:
Well, both are the case, actually.  It's not clear when that support will be merged, and it's not clear that that support is actually the way forward.  For example, porting Electrum and BitcoinJ to Namecoin has already been started, and they may win out over the NMControl-based API.  DNSSEC+DNSCrypt may also be a more standard alternative to an API REST server in cases where DNS users are willing to trust a server without doing SPV verification.  Given that these things are still up in the air and there is not a clear consensus on them at this point, I don't think it's wise to mention them in end-user documentation.  (For development notes, mentioning them is perfectly fine, as long as the above caveats/uncertainties are also mentioned.)